contracts/rem.auth/src/rem.auth.cpp

```diff
@@ -4,52 +4,57 @@
 
 #include <rem.auth/rem.auth.hpp>
 #include <rem.system/rem.system.hpp>
-#include <rem.swap/rem.swap.hpp>
 #include <rem.oracle/rem.oracle.hpp>
 #include <rem.token/rem.token.hpp>
 
 namespace eosio {
    using eosiosystem::system_contract;
 
-   void auth::addkeyacc(const name &account, const string &pub_key_str, const signature &signed_by_pub_key,
-                        const string &extra_pub_key, const asset &price_limit, const string &payer_str)
+   void auth::addkeyacc(const name &account, const public_key &pub_key, const signature &signed_by_pub_key,
+                        const asset &price_limit, const string &payer_str)
```

Comment:
let's use **payer** with the **name** type instead of **payer_str**?

Reply:
@roman-tik, done.